<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>810</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1260</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1260</width>
    <height>810</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>5</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>5</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>580</y>
      <width>280</width>
      <height>158</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_w">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>600</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(w)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_s">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>660</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(s)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_a">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>630</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(a)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_d">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>630</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(d)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_f">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>650</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>上(f)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_c">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>680</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>左(c)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_b">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>680</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>右(b)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_v">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>710</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>下(v)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rl">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>680</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(,)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rr">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>680</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(/)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rt">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>650</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(;)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rb">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>710</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(.)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_j">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>630</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>Y(j)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_i">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>600</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>X(i)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_l">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>630</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>A(l)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_k">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>660</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>B(k)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>590</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>+(y)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_r">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>590</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>-(r)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_h">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>620</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>H(h)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_g">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>620</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>C(g)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>635</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(x)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rc">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>685</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>(')</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_e">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>600</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>L(e)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_q">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>570</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>ZL(q)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_u">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>600</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>R(u)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_o">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>570</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(209, 209, 209)</string>
    </property>
    <property name="text">
     <string>ZR(o)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_action">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>740</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>实时命令：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>281</width>
      <height>721</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
